addons/hr/models/res_users.py

```diff
@@ -1,21 +1,156 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, models, fields
+from odoo import api, models, fields, _
+from odoo.exceptions import AccessError
 
 
 class User(models.Model):
+
     _inherit = ['res.users']
 
-    employee_ids = fields.One2many('hr.employee', 'user_id', string='Related employees')
+    # note: a user can only be linked to one employee per company
+    employee_ids = fields.One2many('hr.employee', 'user_id', string='Related employee')
+    employee_id = fields.Many2one('hr.employee', string="Company employee", compute='_compute_company_employee', store=True)
+
+    job_title = fields.Char(related='employee_id.job_title', readonly=False)
```

Comment:
"So on the user form, you'll get the information from the first employee linked to it...

Is it intended ? "

EDIT: While I was writing this, you applied the comment, wow.

Reply:
So much proactivity